chanzuckerberg/single-cell-data-portal
# chore: 1210 Add Plausible

### Reviewers
**Functional:** 
@seve 

**Readability:** 
@dimple-jethani 

---

## Changes
1. Add Plausible script
2. Update CSP to allow Plausible domain
3. Update config files to include Plausible env. Only prod.js is prod env, everything else uses staging env
4. NOTE: Plausible script ignore `localhost` event, so you'll only see `console.warn()` in the console when running locally

<img width="1393" alt="Screen Shot 2021-09-24 at 1 02 19 PM" src="https://user-images.githubusercontent.com/6309723/134734341-0dfd91d5-a9c9-42cb-88ec-a64276b3de04.png">

Live link: https://thuang-plausible-frontend.rdev.single-cell.czi.technology/

PTAL thank you! 🎉 

## Definition of Done (from ticket)

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1210

## QA steps (optional)
1. Access Plausible Dashboard for staging env: https://plausible.io/cellxgene.staging.single-cell.czi.technology
2. Log in credentials in 1Password `cellxgene` vault
3. Check the current active user count
4. Visit: https://thuang-plausible-frontend.rdev.single-cell.czi.technology/
5. Check current active user count again, should increase by 1 (YOU!)

## Known Issues
